### ISSUE TITLE
Tracking issue for PT2 dashboard training passrate

### ISSUE THREAD

**Issue by shunting314** [pytorch/pytorch#120129]
This issue tracks the PT2 dashboard training pass rate (excluding dynamic shape). The pass rate is already pretty high so the issue just list the failed models and track the progress of fixing them.

# TB

- [x] yolov3: fail with eager_two_runs_differ in cudagraphs setting. The error is transient. It passes for the last 7 days. We should keep watching it for a bit longer (2024 Feb22).
- [ ] vision_maskrcnn: fail with 'fail_accuracy' in default/cudagraphs setttings; fail with 'fail_to_run' in maxautotune setting.
- [x] torchrec_dlrm: fail with 'fail_accuracy' in maxautotune setting. https://github.com/pytorch/pytorch/pull/122012
- [ ] hf_Whisper: fail with 'fail_accuracy' in maxautotune setting. This one is flaky [link](https://hud.pytorch.org/benchmark/torchbench/inductor_max_autotune?startTime=Fri,%2001%20Mar%202024%2022:39:21%20GMT&stopTime=Fri,%2015%20Mar%202024%2021:39:21%20GMT&granularity=hour&mode=training&model=hf_Whisper&dtype=amp&lBranch=main&lCommit=660ec3d38d9d1c8567471ae7fe5b40ae7c6d7438&rBranch=main&rCommit=f351a71dbb68fa4a595e23e8c4be41e32a317742). Can not repro locally
- [x] tts_angular: pass accuracy test but fail performance run in cudagraphs/maxautotune settings. https://github.com/pytorch/pytorch/issues/120474

# HF

- [ ] DebertaForQuestionAnswering: fail with 'fail_accuracy' in maxautotune setting. Fail quite consistently on the dashboard [link](https://hud.pytorch.org/benchmark/huggingface/inductor_max_autotune?startTime=Sat,%2016%20Dec%202023%2022:57:37%20GMT&stopTime=Fri,%2015%20Mar%202024%2021:57:37%20GMT&granularity=day&mode=training&model=DebertaForQuestionAnswering&dtype=amp&lBranch=main&lCommit=b7f25226929e70187a9f36c393665abad0b25190&rBranch=main&rCommit=622947afa86b1c9c3db37466f43984c73db77677), but can not repro locally.

# TIMM

- [ ] mobilenetv2_100: fail with 'fail_accuracy' in maxautotune setting. Fail quite consistently on the dashboard [link](https://hud.pytorch.org/benchmark/timm_models/inductor_max_autotune?startTime=Sat,%2016%20Dec%202023%2023:16:38%20GMT&stopTime=Fri,%2015%20Mar%202024%2022:16:38%20GMT&granularity=day&mode=training&model=mobilenetv2_100&dtype=amp&lBranch=main&lCommit=b7f25226929e70187a9f36c393665abad0b25190&rBranch=main&rCommit=622947afa86b1c9c3db37466f43984c73db77677), but can not repro locally

# Tracking regressions
- [ ] DebertaForQuestionAnswering and nanogpt regression. https://github.com/pytorch/pytorch/issues/122987

cc @ezyang @msaroufim @bdhirsh @anijain2305 @zou3519 @penguinwu @eellison

**Comment by masnesral** [comment]
[old issue scrubbing] All the models listed in the summary look they're all passing as of today, so I'm assuming this issue is just stale. Closing it out.
